tribomobile/screens/SideNavScreen.js

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import   {TouchableOpacity, Text, View} from 'react-native'
+
+import MenuHide from '../components/menuHide'
+import SideBarOption from '../components/SideBarOption'
+
+const ClicableLink = () => {
+  return(
+    <TouchableOpacity style={{alignItems: 'center', marginTop: 85}}>
+      <Text style={{textDecorationLine:'underline', color: 'gray'}}>Salir de mi cuenta</Text>
+    </TouchableOpacity>
+  );
+}
+
+
+const SideNavScreen = ()=>{
+  return(
+    <>   
+      <TouchableOpacity>
+        <MenuHide/>
+      </TouchableOpacity>    
+      <View style={{marginTop: '40%'}}>
+        <SideBarOption
+           textSideBar={'Regístrate/Mi cuenta'}
+        />
+        <SideBarOption
+          textSideBar={'#QuédateEnCasa' }
+          textSideBar2={'Recomendaciones para \nel asilamiento preventivo'}
+        />
+        <SideBarOption
+          textSideBar={'Términos y condiciones '}
+        />
+        <SideBarOption
+          textSideBar={'FAQ'}
+        />
+        <SideBarOption
+          textSideBar={'Políticas de privacidad'}
+        />
```

Comment:
I would say this can be improved, by using an array with all those text, iterate it and  return the component.
```
const arrayexample = [
 "Recomendaciones para \nel asilamiento preventive",
  'Términos y condiciones ',
  'etc',
]


return arrayexample.map(item => {
  <SideBarOption
    textSideBar={item}
   />
}